src/jdb_to_nwb/convert_raw_ephys.py

```diff
@@ -233,14 +252,20 @@ def get_port_visits(folder_path: Path):
     # For reference, a typical port visit keeps the channel high for ~10ms, so 10 samples at 1000 Hz
 
     # Memory-map the large .dat file to avoid loading everything into memory. Reshape to (samples, channels)
-    file_path = folder_path + "/experiment1/recording1/continuous/Rhythm_FPGA-100.0/continuous.dat"
+    # file_path will be something like "/experiment1/recording1/continuous/Rhythm_FPGA-100.0/continuous.dat"
+    pattern = os.path.join(folder_path, "experiment*/recording*/continuous/*/continuous.dat")
+    file_path = glob.glob(pattern)[0] 
```

Comment:
I recommend asserting that the length of the glob is 1. I think there could be multiple recordings in the same root folder accidentally if someone stops and restarts the recording.